defi/src/protocols/data3.ts

```diff
@@ -45471,6 +45471,29 @@ const data3: Protocol[] = [
     oracles: [],
     listedAt: 1717725054
   },
+  {
+    id: "4738",
+    name: "MoveGPT Launchpad",
+    address: "aptos:0x63be1898a424616367e19bbd881f456a78470e123e2770b5b5dcdceb61279c54::movegpt_token::MovegptCoin",
+    symbol: "MGPT",
+    url: "https://movegpt.io/",
+    description:
+      "THE MOST TRUSTED.",
+    chain: "Aptos",
+    logo: `${baseIconsUrl}/movegpt.png`,
+    audits: "2",
+    audit_note: null,
+    gecko_id: "movegpt",
+    cmcId: null,
+    category: "Dexes",
+    chains: ["Aptos"],
+    module: "movegpt/index.js",
```

Comment:
this doesn't exist https://github.com/DefiLlama/DefiLlama-Adapters/blob/main/projects/movegpt/index.js

Reply:
we submited https://github.com/DefiLlama/DefiLlama-Adapters/pull/10547, please help me check

Reply:
We are update my MGPT token, please check

Reply:
Please check , mate